contribute/backend/package-hierarchy.md

```diff
@@ -70,34 +69,32 @@ regular service without the need of complicated setup.
 
 ### Separate store and logic
 
-When building a new service, data validation, manipulation, scheduled
-events and so forth should be collected in a service implementation that
-is built to be agnostic about its store.
+When building a new service, collect data validation, manipulation, scheduled
+events, and so forth, in a service implementation. This implementation should
+be built so that it is agnostic about its store.
 
 The storage should be an interface that is not directly called from
 outside the service and should be kept to a minimum complexity to
 provide the functionality necessary for the service.
 
-A litmus test to reduce the complexity of the storage interface is
-whether an in-memory implementation is a feasible test double to build
-to test the service.
+Usa a simple litmus test to determine whether an in-memory implementation is a feasible test-double to assess the service. This will reduce the complexity of the storage interface. 
```

Comment:
```suggestion
Use a simple litmus test to determine whether an in-memory implementation is a feasible test-double to assess the service. This will reduce the complexity of the storage interface. 
```